okapi-core/src/main/java/org/folio/okapi/util/TcpPortWaiting.java

```diff
@@ -34,24 +34,26 @@ public TcpPortWaiting(Vertx vertx, String host, int port) {
   }
 
   private Future<Void> tryConnect(NuProcess process, int count) {
-    NetClientOptions options = new NetClientOptions().setConnectTimeout(MILLISECONDS);
-    NetClient c = vertx.createNetClient(options);
+    WebClientOptions options = new WebClientOptions().setConnectTimeout(MILLISECONDS);
```

Comment:
```suggestion
    // don't use NetClient because container ports are immediately ready, instead check for HTTP response
    WebClientOptions options = new WebClientOptions().setConnectTimeout(MILLISECONDS);
```